components/contact-details.tsx

```diff
@@ -11,6 +11,20 @@ type ContactDetailsProps = {
   provinceName: string;
 };
 
+// ? please don't remove this ReportButton until issue is resolved. - @kalwabed
+// reference: https://github.com/kawalcovid19/wargabantuwarga.com/pull/171
+// const ReportButton = (props: ReportButtonProps) => (
```

Comment:
If we remove this, the style will break. Mas @zainfathoni, maybe this is an issue from the JIT Engine? I don't know. 😅

Reply:
Probably 😅

Let's keep it for now and try to resolve it separately. 😁